docs/articles_en/get_started/installing-openvino-overview/installing-openvino-linux-header/installing-openvino-zypper.rst

```diff
@@ -0,0 +1,196 @@
+.. {#openvino_docs_install_guides_installing_openvino_zypper}
+
+Install OpenVINO™ Runtime on Linux From ZYPPER Repository
+========================================================
+
+
+.. meta::
+   :description: Learn how to install OpenVINO™ Runtime on Linux operating
+                 system, using the ZYPPER repository.
+
+.. note::
+
+   Note that the ZYPPER distribution:
+
+   * offers both C/C++ and Python APIs
+   * does not offer support for NPU inference
+   * is dedicated to Linux users only
+   * additionally includes code samples
+
+.. tab-set::
+
+   .. tab-item:: System Requirements
+      :sync: system-requirements
+
+      | Full requirement listing is available in:
+      | :doc:`System Requirements Page <system_requirements>`
+
+      .. note::
+
+         OpenVINO RPM packages are compatible with and can be run on the following operating systems:
+
+         - openSUSE 15.4 and openSUSE 15.5
```

Comment:
is it possible to build the same packages for ARM64 architecture?

Reply:
yes but roadmap is:
- Officially publish to distribution (in progress)
- Work on SUSE Entrerprise distribution
- Porting to the ARM platform (in progress)
   + evidence:
   + https://build.opensuse.org/projects/home:cabelo:intel/packages/openvino/repositories/openSUSE_Tumbleweed/binaries
   + https://build.opensuse.org/projects/home:cabelo:intel/packages/openvino/repositories/15.5/binaries

@ilya-lavrenov  is it OK for you?

Reply:
yes, looks great